publish/publish-snapshot.sh

```diff
@@ -111,20 +111,26 @@ fi
 for pom in ${pomFiles}; do
   pom_dir="$(dirname "${pom}")"
   for FILE in "${pom_dir}"/*;  do
-    if [[ ! $FILE == *"-javadoc.jar"* ]] && [[ ! $FILE == *"-sources.jar"* ]]; then
+    if [[ $FILE == $pom ]] ; then
+      continue
+    fi
```

Comment:
This might be nitpick but isnt if file != $pom is better logic?
Or the small if loop is prefered here?

Reply:
Updated and added a comment why this is skipped.